A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js

```diff
@@ -690,14 +689,33 @@ class A320_Neo_CDU_MainDisplay extends FMCMainDisplay {
         if (this.currentFlightPhase <= 2 && SimVar.GetSimVarValue("L:A32NX_Preflight_Complete", "Bool") == 1) {
             if (!(highestThrottleDetent == ThrottleMode.TOGA || highestThrottleDetent == ThrottleMode.FLEX_MCT) && SimVar.GetSimVarValue("RADIO HEIGHT", "Feet") < 100) {
                 SimVar.SetSimVarValue("L:A32NX_Preflight_Complete", "Bool", 0);
+                this.climbTransitionGroundAltitude = null;
             }
         }
-
+        
         //Changes to climb phase when acceleration altitude is reached
         if (this.currentFlightPhase === FlightPhase.FLIGHT_PHASE_TAKEOFF && airSpeed > 80) {
-            const agl = Simplane.getAltitudeAboveGround();
-            if (agl > (this.accelerationAltitude || this.thrustReductionAltitude || 1500)) {
+            const planeAltitudeMsl = Simplane.getAltitude();
+            let accelerationAltitudeMsl = (this.accelerationAltitude || this.thrustReductionAltitude);
+            
+            if(!accelerationAltitudeMsl) {     
+                if(!this.climbTransitionGroundAltitude) {
+                    let airport = this.flightPlanManager.getOrigin();
```

Comment:
`const airport` since value is not changed.

Reply:
Fixed in next commit